core/templates/pages/community-dashboard-page/community-dashboard-page.component.spec.ts

```diff
@@ -147,7 +146,7 @@ describe('Community dashboard page', function() {
       $rootScope.$apply();
     });
 
-    it('should have no username', function() {
+    it('should have no user data in dashboard page', function() {
```

Comment:
when? Is it when the user is not logged in?

Reply:
Yes, this situation is described in spec's `describe` block.